proto/prysm/v1alpha1/wrapper/beacon_block_altair.go

```diff
@@ -86,6 +87,13 @@ func (w altairSignedBeaconBlock) Proto() proto.Message {
 	return w.b
 }
 
+// PbGenericBlock returns a generic signed beacon block.
+func (w altairSignedBeaconBlock) PbGenericBlock() (*eth.GenericSignedBeaconBlock, error) {
```

Comment:
Please add unit tests. Same for Bellatrix and phase0

Reply:
Done!